middleware/http/client_test.go

```diff
@@ -0,0 +1,52 @@
+package http_test
+
+import (
+	"net/http"
+	"testing"
+
+	zipkin "github.com/openzipkin/zipkin-go"
+	httpmiddleware "github.com/openzipkin/zipkin-go/middleware/http"
+	"github.com/openzipkin/zipkin-go/reporter/recorder"
+)
+
+func TestMiddleware(t *testing.T) {
+	reporter := recorder.NewReporter()
+	defer reporter.Close()
+
+	ep, _ := zipkin.NewEndpoint("httpClient", "")
+	tracer, err := zipkin.NewTracer(reporter, zipkin.WithLocalEndpoint(ep))
+	if err != nil {
+		panic(err)
+	}
+
+	client, err := httpmiddleware.NewClient(tracer, nil, httpmiddleware.WithHTTPTrace(true))
```

Comment:
not sure possible in go, but an inline comment describing the nil .. ex /* remote endpoint */ or whatever it is that's missing

Reply:
Yes that is possible and i will add comment